### DISCUSSION TITLE
What type of service road is this - needs OSM Tagging Proposal

### DISCUSSION THREAD

**Discussion by dbdean** [streetcomplete/StreetComplete#2838] [Ideas]
Hi all,

I want to have a quest that asks "What type of service road is this?" and indicates as such on the service tag, but it cannot work because there is no service tag for the main service roads in a large facility and/or the main roads of a parking lot. There is a closed issue if you want more info: https://github.com/streetcomplete/StreetComplete/issues/808.

I've made a start in the past in braving the OSM Tagging Proposal process, but I just don't have it in me to go through that process.

Previous discussion on the tagging list is here: https://lists.openstreetmap.org/pipermail/tagging/2020-August/054470.html. I don't think we really came to a solid conclusion. I should have put it up for a vote, but I wasn't sure what the options should have been at the time.

Is anyone who's has a better temperament for dealing with OSM Tagging interested in getting the tagging side fixed so we can get back to implementing it as a quest?

I'm happy to help otherwise - I just don't want to get involved with the OSM Tagging proposal process.

- David

**Comment by dbdean** [comment]
Nuts. I guess it's up to me :(

**Reply by smichel17** [reply]
:reminder_ribbon: *sympathies*
